### DISCUSSION TITLE
Is not working? HTTP 401

### DISCUSSION THREAD

**Discussion by alec1o** [FajarKim/github-readme-profile#349] [Q&A] [answered]
The example isn't working...?
![Screenshot from 2024-05-21 10-27-48](https://github.com/FajarKim/github-readme-profile/assets/100610503/8e4396cd-bf32-43ee-aad9-3f2b61231101)

**Comment by FajarKim** [comment, marked as answer]
Sorry for answering now. I have corrected this. This happened because my GitHub Token API needed to be updated.

**Reply by alec1o** [reply]
Thanks.
